substra/cli/interface.py

```diff
@@ -521,7 +536,8 @@ def add_compute_plan(ctx, tuples):
             "test_data_sample_keys": list[str],
             "traintuple_id": str,
             "tag": str,
-        }]
+        }],
+        "metadata": list[str]
```

Comment:
There are also metadata for compute plan's tuples, they need to be included here as well.

Reply:
Also, metadata is not a list[str]